ansible/roles/rancher/tasks/cert-manager.yml

```diff
@@ -5,9 +5,17 @@
   ignore_errors: true
   changed_when: false
   register: crd
-  
+
+- name: Copy cert-manager Custom Resoure Definintion
+  become: true
+  copy: src="{{ item }}" dest="/etc/kubernetes/{{ item }}" mode=600 owner=root
+  with_items:
```

Comment:
no need for the `with_items` since we only have a single cert file. Just specify it without the loop.

Reply:
fixed in follow up commit.